src/System.Runtime/tests/Performance/Perf.GC.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Xunit.Performance;
+using System.Text;
+
+namespace System.Runtime.Tests
+{
+    public class Perf_GC
+    {
+        [Benchmark]
+        public void SuppressFinalize()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                // obj has no managed resources to free, so suppressing the finalizer for it is fine.
+                object obj = new object();
+
+                using (iteration.StartMeasurement())
+                {
+                    GC.SuppressFinalize(obj); GC.SuppressFinalize(obj); GC.SuppressFinalize(obj);
+                    GC.SuppressFinalize(obj); GC.SuppressFinalize(obj); GC.SuppressFinalize(obj);
+                    GC.SuppressFinalize(obj); GC.SuppressFinalize(obj); GC.SuppressFinalize(obj);
```

Comment:
Is the cost of SuppressFinalize on an object that's already had finalization suppressed the same as on one that hasn't?  I wouldn't be surprised if the answer is no, in which case 8 of these 9 calls would behave differently from what I think you're trying to test.

Additionally, System.Object doesn't actually have a finalizer (even though it exposes the virtual Finalize method), so there's nothing to suppress here.  I imagine you'd instead want to define your own helper class, e.g.
```C#
private class ObjectWithFinalizer
{
    ~ObjectWithFinalizer() { }
}
```
and use that instead of Object.